app/server/lib/OIDCConfig.ts

```diff
@@ -69,6 +77,14 @@ export class OIDCConfig {
       envVar: 'GRIST_OIDC_IDP_CLIENT_SECRET',
       censor: true,
     });
+    this._namePropertyKey = section.flag('namePropertyKey').readString({
+      envVar: 'GRIST_OIDC_SP_PROFILE_NAME_ATTR',
+    });
+
+    this._emailPropertyKey = section.flag('emailPropertyKey').readString({
```

Comment:
With `defaultValue`, you should be able to change to `requireString`, and the result would have type `string` (so you wouldn't need the `?` in declaration, or `!` in usage).